src/scss/settings/_variables.scss

```diff
@@ -38,12 +38,16 @@ $type: (
         narrow: (16, 21)
     ),
     large--scaleUp: (
-        default: (28, 36),
+        default: (32, 32),
```

Comment:
Same here - can't imagine this is intentional as the space between lines of text is very shallow